finagle-mysql/src/test/scala/com/twitter/finagle/mysql/ParameterTest.scala

```diff
@@ -0,0 +1,28 @@
+package com.twitter.finagle.mysql
+
+import com.twitter.finagle.exp.mysql.Parameter
+import com.twitter.finagle.exp.mysql.Parameter.NullParameter
+import org.junit.runner.RunWith
+import org.scalatest.FunSuite
+import org.scalatest.junit.JUnitRunner
+
+@RunWith(classOf[JUnitRunner])
+class ParameterTest extends FunSuite {
+  test("Parameter coercion") {
+    locally {
```

Comment:
instead of locally blocks, can we just separate these out into clearly named tests?  that will help us understand what a test failure means if it happens.